pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -452,27 +453,41 @@ func setAvailableProgressingNodeInstallerFailingConditions(newStatus *operatorv1
 
 	revisionStrings := []string{}
 	for _, currentRevision := range Int32KeySet(counts).List() {
-		count := counts[currentRevision]
-		revisionStrings = append(revisionStrings, fmt.Sprintf("%d nodes are at revision %d", count, currentRevision))
+		if count := counts[currentRevision]; count == 1 {
+			revisionStrings = append(revisionStrings, fmt.Sprintf("1 node achieved revision %d", currentRevision))
```

Comment:
is "achieved" the right word?

Reply:
we should pick one and be consistent "are at" vs. "achieved" I don't care much which one, I liked "achieved" better.